HW4_modules/filterseq.py

```diff
@@ -0,0 +1,33 @@
+def gc_cont(sequence: str) -> float:

```

Comment:
Чтобы было глаголом я бы скорее сказал в духе
```suggestion
def calculate_gc(sequence: str) -> float:

```